### PR TITLE
Adding score points

### PR DESCRIPTION
**Pull request by kimjetam**

### CODE REVIEW

File: src/renderer/App.tsx [hunk 1]
```diff
@@ -102,14 +116,22 @@ export default function App() {
   // game (back button or home button), team users should be navigated to their home screen too. The 'quitGame' message is send to them for
   // this purpose.
   router.subscribe((x) => {
+    // case when you exit game-host screen without finishing game
     if (
       prevLocationRoute &&
       prevLocationRoute === RouteDefinitions.GameHost.route &&
       x.location.pathname !== RouteDefinitions.GameSettings &&
+      x.location.pathname !== RouteDefinitions.GameFinished &&
```
Comment (podhorets) [inline src/renderer/App.tsx:124]: should this IF be extended every time we define new routes?

File: src/renderer/App.tsx [hunk 2]
```diff
@@ -102,14 +116,22 @@ export default function App() {
...
       !x.matches.some((m) => m.route.path === RouteDefinitions.GameHost.route)
     ) {
       sendMessage('quitGame', currentUser?.id);
     }
+
+    // case when you finish game and exit GameFinished screen
+    if (prevLocationRoute && prevLocationRoute === RouteDefinitions.GameFinished) {
+      sendMessage('quitGame', currentUser?.id);
```
Comment (podhorets) [inline src/renderer/App.tsx:132]: same as on line 127, you split it just to understand the cases we have ?

**Review by podhorets — APPROVED** [review]

File: src/renderer/App.tsx (hunk 2 above)
Comment (kimjetam) [reply src/renderer/App.tsx:132]: yes I split it to be little bit more readable

File: src/renderer/App.tsx (hunk 1 above)
Comment (kimjetam) [reply src/renderer/App.tsx:124]: The logic inside router.subscribe is used to detect, if the running game was closed. Imagine team screens. They first see Welcome Team page. And as soon as host starts a game, they see "wait for next question" screen. However, if host closes the game (by navigating back, or clicking home button), or the game finishes, we want team users to redirect back to their home screen (welcome team). For that we are using this send message 'quitGame'. We are storing previous screen on host, and comparing it with current screen, by this we know if he cancelled or finished the game. So this logic needs to be adapted only if we add new screen to the running game.

**Pull request merged**